lib/matplotlib/figure.py

```diff
@@ -2155,8 +2155,8 @@ def __init__(self, parent, subplotspec, *,
             Defines the region in a parent gridspec where the subfigure will
             be placed.
 
-        facecolor : default: :rc:`figure.facecolor`
-            The figure patch face color.
+        facecolor : default: ``"none"``
+            Transparent face color.
```

Comment:
```suggestion
            The figure patch face color; transparent by default.
```
or similar.  The point is that it could be any colour.

Reply:
I think you committed this suggestion, but then it got clobbered in the git confusion.  Easily done.

Reply:
You are right. Sorry about that. I amended the commit to add it back in.

Reply:
No worries - I've done similar!